nodes/logic/custom_switcher.py

```diff
@@ -104,6 +104,13 @@ def process(self):
 
         self.outputs['Item'].sv_set([[i for i, b in enumerate(self.user_list) if b]])
 
+    def load_from_json(self, node_dict: dict, import_version: float):
+        '''function to get data when importing from json'''
```

Comment:
Using such methods should be avoided by any cost. I mentioned about that when I was refactoring JSON serialization code.